README.md

```diff
@@ -425,7 +437,13 @@ If `topic` is something else then `payload` must be an object and tells both the
 
 **Local Authentication**
 
-  `Username` and `Password`: A username and password used when you link Google SmartHome to this node.
+  `Use Google Login`: If enabled, use the Google login authentication.
+
+  `Login Client ID`: If Google Login is enabled, The client id you gained from the *Google Sign-In* integration..
+
+  `Authorized emails`: If Google Login is enabled, The email, comma-separated, authorized to log in.
```

Comment:
Still says "comma-separated" here.

Reply:
Sorry again, I've changed the locales but I forgot the README.